clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -727,6 +730,16 @@ private void checkGroupRemoteAssignor() {
         }
     }
 
+    private void checkPartitionAssigmentStrategy() {
+        List<String> assignmentStrategies = getList(PARTITION_ASSIGNMENT_STRATEGY_CONFIG);
+        if (getString(GROUP_PROTOCOL_CONFIG).equalsIgnoreCase(GroupProtocol.CONSUMER.name())) {
+            List<Class<? extends AbstractPartitionAssignor>> defaultValue = Arrays.asList(RangeAssignor.class, CooperativeStickyAssignor.class);
```

Comment:
Could you please create a default value of `Arrays.asList(RangeAssignor.class, CooperativeStickyAssignor.class)`? That can keep the consistency in the futrue